test/main.sh

```diff
@@ -616,6 +616,12 @@ run_test test_snap_restore "snapshot restores"
 run_test test_config_profiles "profiles and configuration"
 run_test test_config_edit "container configuration edit"
 run_test test_config_edit_container_snapshot_pool_config "container and snapshot volume configuration edit"
+run_test test_metadata_show "container metadata show"
+run_test test_metadata_edit "contaimer metadata edit"
+run_test test_metadata_template_list "contaimer template list"
+run_test test_metadata_template_content "contaimer template content"
+run_test test_metadata_template_add "container template add"
+run_test test_metadata_template_remove "container template remove"
```

Comment:
This ought to be a single entry. The entry can call the sub-functions though.

The problem with adding a lot of "run_test" is that it creates as many sections on Jenkins and makes it a bit harder for me to go through things.